web/modules/custom/do_base/css/blog-rss-feed.css

```
@@ -0,0 +1,30 @@
+  justify-content: center;
+  padding: 1.5rem 0;
+}
+
+.do-blog-rss__link {
+  display: inline-flex;
+  align-items: center;
+  gap: 0.4em;
+  font-size: 0.875rem;
+  color: #e8620e;
+  text-decoration: none;
+  border: 1px solid currentColor;
```

Stylelint (17.3.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)